pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00_test.go

```diff
@@ -280,9 +280,9 @@ func TestAvailableStatus(t *testing.T) {
 			},
 		},
 		{
-			name:           "MultipleAPIServiceNotAvailable",
-			expectedStatus: operatorv1.ConditionFalse,
-			expectedReason: "Multiple",
+			name:            "MultipleAPIServiceNotAvailable",
+			expectedStatus:  operatorv1.ConditionFalse,
+			expectedReasons: []string{"APIServiceNotAvailable1", "APIServiceNotAvailable2"},
```

Comment:
These don't get indexed in the code so this will never pass. Omit the numbers.

Reply:
Fixed.